.github/workflows/build_container_latest.yml

```diff
@@ -1,31 +1,31 @@
-name: Publish container image
+name: Build latest container image
 
 on:
   push:
     branches:
       - main
+    tags:
```

Comment:
can we remove the tags from the run trigger?

Reply:
Yup, just fixed